compiler/scenario-service/client/src/DA/Daml/LF/PrettyScenario.hs

```diff
@@ -829,10 +833,14 @@ renderRow world parties NodeInfo{..} =
             , H.th "status"
             , ths
             ]
-        observed party = if party `S.member` niObservers then "X" else "-"
+        viewStatus party
+            | party `S.member` niSignatories = "S"  -- "S" for signatory.
+            | party `S.member` niStakeholders = "O"  -- "O" for observer.
+            | party `S.member` niWitnesses = "D"  -- "D" for disclosed/divulged.
+            | otherwise = "-"
         active = if niActive then "active" else "archived"
         row = H.tr H.! A.class_ (H.textValue active) $ mconcat
-            [ foldMap ((H.td H.! A.class_ "disclosure") . H.text . observed) parties
+            [ foldMap ((H.td H.! A.class_ "disclosure") . H.text . viewStatus) parties
```

Comment:
How about we display that at the end instead of at the beginning? If I have 50 parties that becomes completely unusable otherwise.

Reply:
This PR has not really changed the situation here. But since I agree and it's easy enough to fix, I've moved them to the end.